fiftyone/core/metadata.py

```diff
@@ -283,7 +283,7 @@ def get_image_info(f):
     Returns:
         ``(width, height, num_channels)``
     """
-    img = Image.open(f)
+    img = ImageOps.exif_transpose(Image.open(f))
```

Comment:
Since this is an extremely core function call, I would just want to be careful to not add any latency if not necessary.
```suggestion
    img = Image.open(f)
    if 'exif' in img.info:
        img = ImageOps.exif_transpose(img)
```

Reply:
Guess I never hit submit on that comment

Reply:
That's a good call, thanks for the change. It's an extremely busy period for me so I haven't been able to look at adding a test function, not sure when I'll be able to look into that but hopefully within a couple of weeks I can give a look at that :)

Reply:
No worries I'm working on it now so we can get this out with the next release :)